project/app/controllers/predictionController.js

```diff
@@ -2,7 +2,10 @@ var Prediction = require('../models/prediction');
 
 var getPredictScore = function (gre, toefl, res) {
     // return random value as prediction for now (prototyping)
-    res.send((Math.random() * 100).toString().slice(0, 4));
+    res.json({
+        status: "OK",
```

Comment:
status better be changed to one bit/byte for portability. in this case an integer 0/1 should suffice. 0 means success and 1 means error.

reason being networking roundtrip is expensive.